inf-clojure.el

```diff
@@ -131,6 +134,10 @@ The following commands are available:
 
 \\{inf-clojure-minor-mode-map}"
   :lighter "" :keymap inf-clojure-minor-mode-map
+  (inf-clojure-eldoc-setup)
+  ;; No ElDoc support for ClojureScript yet
+  (unless (equal major-mode 'clojurescript-mode)
```

Comment:
It's generally better to use `derived-mode-p`, as it will work with derived modes as well.